scripts/bigshot.lic

```diff
@@ -2691,8 +2693,12 @@ class Bigshot
         next
       elsif result =~ /may not be activated within 60 seconds of a Multi-Strike\./i
         break
+      elsif result =~ /(?:Barrage|Flurry|Fury|Guardant Thrusts|Pummel|Thrash) is still in cooldown\./i

```

Comment:
If you hit a race where Effects thinks the cooldown is done but you race with the game's more precise check you won't retry so it would skip the command